server/test/views/components/PageNotProductionBannerTest.java

```diff
@@ -4,68 +4,115 @@
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
+import java.util.Collections;
+import java.util.Map;
 import java.util.Optional;
 import org.junit.Before;
 import org.junit.Test;
+import play.i18n.Lang;
+import play.i18n.Langs;
+import play.i18n.Messages;
+import play.i18n.MessagesApi;
 import play.mvc.Http;
 import services.DeploymentType;
+import services.MessageKey;
 import services.settings.SettingsManifest;
 
 public class PageNotProductionBannerTest {
   SettingsManifest settingsManifest;
   Http.Request request;
+  Messages messages;
 
   @Before
   public void setUp() {
     settingsManifest = mock(SettingsManifest.class);
     request = mock(Http.Request.class);
+
+    // Setup messages
+    Langs langs = new Langs(new play.api.i18n.DefaultLangs());
+    Map<String, String> messagesMap = new java.util.HashMap<>();
+    messagesMap.put(MessageKey.NOT_FOR_PRODUCTION_BANNER_LINE_1.getKeyName(), "line1");
+    messagesMap.put(MessageKey.NOT_FOR_PRODUCTION_BANNER_LINE_2.getKeyName(), "line2 {0}");
+
+    Map<String, Map<String, String>> langMap =
+        Collections.singletonMap(Lang.defaultLang().code(), messagesMap);
+    MessagesApi messagesApi = play.test.Helpers.stubMessagesApi(langMap, langs);
+    messages = messagesApi.preferred(langs.availables());
```

Comment:
Don't try to use mockito for mocking messages. You'll have a bad time. This is what Play recommends in their docs.